plugins/occt/module/CMakeLists.txt

```diff
@@ -16,9 +16,12 @@ vtk_module_include(f3d::vtkextOCCT
   PRIVATE
     $<BUILD_INTERFACE:${OpenCASCADE_INCLUDE_DIR}>)
 
-vtk_module_link(f3d::vtkextOCCT
-  PRIVATE
-    ${OpenCASCADE_LIBRARIES})
+set(_occt_link_libs "${OpenCASCADE_LIBRARIES}")
+if(CMAKE_LINK_GROUP_USING_RESCAN_SUPPORTED)
```

Comment:
change the minimum CMake version for `occt` plugin to 3.24

Reply:
I dont think we want to do that. 3.24 is quite new, it is not strictly required to build the occt plugin.

Reply:
And in anycase that check is needed, because rescan is a GCC linker thing only

Reply:
ok but then `CMAKE_LINK_GROUP_USING_RESCAN_SUPPORTED` won't be defined

Reply:
> ok but then `CMAKE_LINK_GROUP_USING_RESCAN_SUPPORTED` won't be defined

That would still work if undefined, but I believe we are checking if variables are undefined?

Reply:
`if(VAR)` is not warning if VAR is not defined.

Reply:
Ok then